aliases/available/docker.aliases.bash

```diff
@@ -0,0 +1,16 @@
+cite 'about-alias'
+about-alias 'docker abbreviations'
+
+alias dklc='docker ps -l'  # List last Docker container
+alias dklcid='docker ps -l -q'  # List last Docker container ID
+alias dklcip="docker inspect `dklcid` | grep IPAddress | cut -d '\"' -f 4"  # Get IP of last Docker container
+alias dkps='docker ps'  # List running Docker containers
+alias dkpsa='docker ps -a'  # List all Docker containers
+alias dki='docker images'  # List Docker images
+alias dkrmac='docker rm $(docker ps -a -q)'  # Delete all Docker containers
+alias dkrmlc='docker-remove-most-recent-container'  # Delete most recent (i.e., last) Docker container
```

Comment:
+1 on consistency.  I've always leaned toward dashes so I'd be happy to consolidate around that, but am open for either.  Maybe we should consider creating a deprecation wrapper and wrap one or the other with warnings?